src/main/java/serene/Ui.java

```diff
@@ -0,0 +1,28 @@
+package serene;
+
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner scanner;
+
+    public Ui() {
+        scanner = new Scanner(System.in);
+    }
+
+    public void showWelcome() {
+        System.out.println("Hello! I'm Serene\nWhat can I do for you?");
+    }
+
+    public void exitMessage() {
```

Comment:
It seems that the naming scheme between `showWelcome` and `exitMessage` here is a little inconsistent, perhaps it would be better for `exitMessage` to a verb instead of a noun?